src/main/java/com/adobe/abp/regola/rules/NumberRule.java

```diff
@@ -58,11 +62,25 @@ Result check(V fact) {
                 return checkFact(fact, (f, value) -> comparator.compare(f, getValue()) < 0);
             case LESS_THAN_EQUAL:
                 return checkFact(fact, (f, value) -> comparator.compare(f, getValue()) <= 0);
+            case MODULO:
+                return checkFact(fact, (f, value) -> f.doubleValue() % getValue().doubleValue() == 0);
```

Comment:
Modulo arithmetic and divisibility is primarily allowed for integers only (there are ways to perform mod on reals, but it is a bit niche as far as I am aware). 

So while `%` will work on reals (double, float in Java), it is not as conventional in mathematics and a bit surprising.

Reply:
So should we explicitly restrict the DIVISIBLE_BY operation to integer types?

Reply:
I think so.